src/v2i-hub/ImmediateForwardPlugin/manifest.json

```diff
@@ -5,72 +5,19 @@
+      "key": "ImmediateForwardConfigurations",
+      "default": "[{\"name\": \"East Intersection Cohda\", \"rsuSpec\": \"RSU4.1\",\"address\": \"127.0.0.1\",\"port\": 3745,\"txMode\": \"CONT\",\"signMessage\": false,\"messages\": [{ \"tmxType\": \"SPAT-P\", \"sendType\": \"SPAT\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"MAP-P\", \"sendType\": \"MAP\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"PSM-P\", \"sendType\": \"PSM\", \"psid\": \"0x27\", \"channel\": 183 }, { \"tmxType\": \"TIM\", \"sendType\": \"TIM\", \"psid\": \"0x8003\", \"channel\": 183 },{ \"tmxType\": \"TMSG07-P\", \"sendType\": \"TMSG07\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"TMSG03-P\", \"sendType\": \"TMSG03\", \"psid\": \"0xBFEE\", \"channel\": 183 },{ \"tmxType\": \"TMSG05-P\", \"sendType\": \"TMSG05\", \"psid\": \"0x8003\", \"channel\": 183 },{ \"tmxType\": \"SSM-P\", \"sendType\": \"SSM\", \"psid\": \"0x8002\", \"channel\": 183 },{ \"tmxType\": \"SDSM\", \"sendType\": \"SDSM\", \"psid\": \"0x8010\", \"channel\": 183 }]}]",
       "description": "JSON data defining the message types, PSIDs, and channel number for messages forwarded to the V2X radio at destination 1."
-    },
-    {
-      "key": "Messages_Destination_2",
-      "default": "{ \"Messages\": [ ] }",
-      "description": "JSON data defining the message types, PSIDs, and channel number for messages forwarded to the V2X radio at destination 2."
-    },
-    {
-      "key": "Messages_Destination_3",
-      "default": "{ \"Messages\": [ ] }",
-      "description": "JSON data defining the message types, PSIDs, and channel number for messages forwarded to the V2X radio at destination 3."
-    },
-    {
-      "key": "Messages_Destination_4",
-      "default": "{ \"Messages\": [ ] }",
-      "description": "JSON data defining the message types, PSIDs, and channel number for messages forwarded to the V2X radio at destination 4."
-    },
-    {
-      "key": "Destination_1",
-      "default": "127.0.0.1:1516",
-      "description": "The destination UDP server(s) and port number(s) on the V2X radio for all messages specified by Messages_Destination_1."
-    },
-    {
-      "key": "Destination_2",
-      "default": "0",
-      "description": "The destination UDP server(s) and port number(s) on the V2X radio for all messages specified by Messages_Destination_2."
-    },
-    {
-      "key": "Destination_3",
-      "default": "0",
-      "description": "The destination UDP server(s) and port number(s) on the V2X radio for all messages specified by Messages_Destination_3."
-    },
-    {
-      "key": "Destination_4",
-      "default": "0",
-       "description": "The destination UDP server(s) and port number(s) on the V2X radio for all messages specified by Messages_Destination_4."
-    },
-    {
-      "key": "TxMode",
```

Comment:
The original design seems to apply the TxMode, signMessage , HSMurl and EnableHSM to all destinations. 

In the new design above seems to apply these parameters to one destination only. If we need to apply the same parameters across all destinations, we need to duplicate these parameter entries in the new config?

Reply:
Yes, these are radio dependent. In other words each radio connection is now able to independently configure TxMode, whether it wants to sign messages and any HSM setup. In NTCIP 1218 there are radio configurations for signing messages that have to be setup for each RSU. 
In any case, I think from the V2X Hub stand point we have decided not to continue to pursue 1609.2 message signing and that this is largely the responsibility of RSUs. The change here does not yet deprecate this functionality. It only makes it configurable for each RSU independently